subcommands/waves/rollout.go

```diff
@@ -83,8 +89,13 @@ func doRolloutWave(cmd *cobra.Command, args []string) {
 	group := readGroup(cmd, args)
 	limit, percentage := readLimit(cmd)
 	uuids := readUuids(cmd)
+	dryRun, _ := cmd.Flags().GetBool("dry-run")
+	printUuids, _ := cmd.Flags().GetBool("print-uuids")
+	printNames, _ := cmd.Flags().GetBool("print-names")
 
-	if len(group) == 0 && len(uuids) == 0 && limit == 0 && percentage == 0 {
+	isFullRollout := len(uuids) == 0 && limit == 0 && percentage == 0
+
+	if len(group) == 0 && isFullRollout {
 		subcommands.DieNotNil(errors.New(
 			"One of the following flags must be set: group, limit, percentage, uuids\n" + cmd.UsageString(),
```

Comment:
The explaination is already a part of the help message, which is BTW printed here.
There is no need to double-tell this to the user.

Reply:
1. The explanation how to use is printed, not a reason of the command failure. Instead it says "error: do something".

It looks like I have to replace all error messages in aklite. So, instead of "err: invalid DG url" aklite should print "err: set proper DG url is sota.toml". :).

Reply:
fioctl is not aklite - they can use different semantics.

Here are some of the fioctl errors generated by cobra for this use case:
- `Error: required flag(s) "role" not set`;
- `Error: if any flags in the group [first-time keys] are set they must all be set; missing [keys]`
- `Error: if any flags in the group [raw diff] are set none of the others can be; [diff raw] were all set`
- `ERROR: One of the following flags must be set: group, limit, percentage, uuids`

The last one is from this PR.  An error here is that the user did not specify one of the required flags. The most I can do to keep it the same look and feel is to add `; none was set` at the end, although I find it redundant.

Also, maybe, we need to change `ERROR:` to `Error:` everywhere; but this is a separate topic.

Reply:
I'm not going to change this. This message goes from the previous PR and is good enough.